README.md

```diff
@@ -12,17 +12,19 @@ SYNOPSIS
 use Image::PNG::Portable;
 my $o = Image::PNG::Portable.new: :width(16), :height(16);
 $o.set: 8,8, 255,255,255;
+$o.set-text-meta('Author', 'Apogee');
```

Comment:
In case you're the changes to the README manually: no need.  App::Mi6 renders the rakudoc on release, or on a `mi6 build`.

Reply:
Thanks, I really need to learn `App::Mi6`.